app/client/src/pages/Editor/APIEditor/PostBodyData.tsx

```diff
@@ -33,6 +33,9 @@ const JSONEditorFieldWrapper = styled.div`
     height: auto;
     min-height: 250px;
   }
+  &.width-65 {
+    width: 65%;
+  }
```

Comment:
Don't define a new class. Select the child directly from here and apply your styles
```suggestion
  ${DynamicAutocompleteInputWrapper} {
    width: 65%;
  }
```